source/lib-intro.tex

```diff
@@ -606,13 +606,13 @@
 The enumerated type \tcode{\placeholder{enumerated}} can be written:
 
 \begin{codeblock}
-enum @\placeholder{enumerated}@ { @$\tcode{\placeholder{V}}_{0}$@, @$\tcode{\placeholder{V}}_{1}$@, @$\tcode{\placeholder{V}}_{2}$@, @$\tcode{\placeholder{V}}_{3}$@, ..... };
+enum @\placeholder{enumerated}@ { @$\tcode{\placeholder{V}}_{0}$@, @$\tcode{\placeholder{V}}_{1}$@, @$\tcode{\placeholder{V}}_{2}$@, @$\tcode{\placeholder{V}}_{3}$@, @$\cdots$@ };
```

Comment:
Are these "center dots"? Why not dots on the baseline (\ldots)? Center dots are for omitted operands of binary folds, like "1 + 2 + \cdots + n".

Reply:
That sounds like (it will have been) an oversight that we should fix. Sure, omit this file from the change and I'll commit the rest.